src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -35,6 +35,12 @@ mappings:
                     mapping:
                         type: 'keyword'
                         normalizer: 'varchar_normalizer'
+            -
+                decimal:
+                    path_match: '*-decimal.*'
```

Comment:
It's great that it works. But I'd have put: 'values.*-decimal.*' can you make the change and see if it works